docs/news/_quarto.yml

```
@@ -20,13 +20,13 @@ website:
       - text: "Guides"
         menu:
           - text: "Workbench Admin Guide"
-            url: 'https://docs.posit.co/ide/server-pro/'
+            url: 'https://docs.posit.co/ide/server-pro/{{< var version>}}'
```

Nitpicking based on the way the quarto docs format variables
```suggestion
            url: 'https://docs.posit.co/ide/server-pro/{{< var version >}}'
```